src/components/MessageLogger/MessageItem/MessageItem.module.scss

```diff
@@ -3,8 +3,11 @@
 .lineMsg {
   width: 100%;
   display: grid;
-  grid-template-columns: 1.2rem auto 1fr 2em;
-  grid-template-areas: "caret id description counter";
+  grid-template-columns: 1.2rem auto auto 1fr;
+  grid-template-rows: 0.9rem auto;
```

Comment:
Does the first row need to have a fixed size? I'd prefer that the row has the size of the content, and because all elements have constant height, every item would have the same height.